Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -66,6 +67,20 @@ def livecheck_formulae(formulae_to_check, args)
       @livecheck_strategy_names.freeze
 
       has_a_newer_upstream_version = false
+
+      if args.json? && !args.quiet? && $stdout.tty?
+        total_formulae = if formulae_to_check == Formula
+          formulae_to_check.count
+        else
+          formulae_to_check.length
+        end
+        ohai "Generating JSON"
```

Comment:
```suggestion
        ohai "Running checks"
```

One last thing, I think. We technically don’t generate the JSON until the end, so “Running checks” is more accurate and also makes it clear what the progress bar is tracking.

Reply:
Haha yes, I always felt the message was not exactly apt. Making the change 👍🏼